src/Utilities/Compiler/Extensions/SymbolVisibility.cs

```diff
@@ -4,7 +4,9 @@
 
 namespace Analyzer.Utilities.Extensions
 {
+#pragma warning disable CA1027 // Mark enums with FlagsAttribute
```

Comment:
Interesting, do we need the values? It would prevent this warning.

Reply:
Possibly not, but I want to avoid any real semantic changes in this PR.